modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -135,6 +164,52 @@ def test_blobFromImage(self):
         target = target.transpose(2, 0, 1).reshape(1, 3, height, width)  # to NCHW
         normAssert(self, blob, target)
 
+    def test_model(self):
+        img_path = self.find_dnn_file("dnn/dog416.png")
+        weights = self.find_dnn_file("dnn/resnet50_rfcn_final.caffemodel")
+        config = self.find_dnn_file("dnn/rfcn_pascal_voc_resnet50.prototxt")
+        frame = cv.imread(img_path)
+        model = cv.dnn.DetectionModel_create(weights, config)
```

Comment:
Can you check that `cv.dnn_DetectionModel(weights, config)` also works? I hope that with https://github.com/opencv/opencv/issues/14730 resolved we can even replace it to `cv.dnn.DetectionModel(weights, config)`.

Reply:
`cv.dnn_DetectionModel(weights, config)` doesn't work.  
 `<class 'cv2.dnn_DetectionModel'> returned NULL without setting an error`
And `cv.dnn.DetectionModel(weights, config)` doesn't work too.  
`AttributeError: module 'cv2.dnn' has no attribute 'DetectionModel'`

Reply:
What if add `CV_WRAP` for `DetectionModel` constructors?